### PR TITLE
[WIP] Workflow for  Quasi-harmonic approximation (forcefields and VASP)

### PR DESCRIPTION
**Pull request by JaGeo**
## Summary

ToDo:

- [x] Reduce number of steps in the workflow
- [x] Finalize Force field tests
- [ ] Check all units and add field descriptions
- [ ] Document the workflows
- [ ] Fix all linting issues
- [ ] write workflows for the different force fields
- [ ] add a VASP Workflow (with minimal settings)

### CODE REVIEW

File: src/atomate2/common/flows/phonons.py [hunk 1]
```diff
@@ -273,6 +273,7 @@ def make(
             )
             jobs.append(static_job)
             total_dft_energy = static_job.output.output.energy
+            stress = static_job.output.output.stress
```
Comment (JaGeo) [inline src/atomate2/common/flows/phonons.py:276]: I will likely just remove this again.

**Comment by JaGeo** [comment]
Hi @janosh do you maybe have any idea where all these errors in unrelated modules could come from? could this be related to a cache?

**Comment by janosh** [comment]
hard to say with so many errors in the CI logs. maybe some breaking change in an indirect dependency? will be interesting to [see if `main` still passes](https://github.com/materialsproject/atomate2/actions/runs/9715112873) today given the last successful run seems to be 4 days old

**Comment by JaGeo** [comment]
@janosh There were updates on other pull requests in the last days. It seemed to work there which confuses me.

**Comment by JaGeo** [comment]
But yes, let's wait for the test on main @janosh . Thanks!

**Comment by JaGeo** [comment]
Nope, it is definitely my branch. I will start fresh again and will copy all files to a new branch and pull request. This is very weird!

**Comment by janosh** [comment]
are your tests passing locally? if yes, that's even stranger. if not, `git bisect` starting from `main` would be a good tool to identify the first failing commit

**Comment by JaGeo** [comment]
@janosh I will try to start with a fresh pull request and add changes step by step. Then, I will see.

**Pull request closed without merging**